components/cart/CartItemActionsMobile/CartItemActionsMobile.tsx

```diff
@@ -16,13 +20,27 @@ const styles = {
     },
     padding: '0.5rem 1rem',
   },
+  cardAction: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'flex-start',
+    p: '0.5rem 0',
+  },
+  button: {
+    minWidth: 'auto',
+    lineHeight: 'initial',
+    color: 'text.primary',
+    justifyContent: 'flex-start',
+    mr: 1,
+  },
 }
 
 const CartItemActionsMobile = (props: CartItemActionsMobileProps) => {
-  const { actions, onMenuItemSelection } = props
+  const { cartItem, actions, onMenuItemSelection, onCartItemDelete } = props
   const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null)
```

Comment:
Provide a meaningful name instead of anchorEl

Reply:
Ignored as per ur confirmation